crates/consensus/authority/src/utils.rs

```diff
@@ -154,3 +154,34 @@ async fn process_botanix_log(
     }
     Ok(())
 }
+
+const MINTING_CONTRACT_ADDRESS: [u8; 20] = hex!("0Ea320990B44236A0cEd0ecC0Fd2b2df33071e78");
```

Comment:
```suggestion
const MINTING_CONTRACT_ADDRESS: [u8; 20] = hex!("0ea320990b44236a0ced0ecc0fd2b2df33071e78");
```

Super nit: consistent letter casing in the hex

Reply:
~~im using the checksum address here which we also use in `mint_validation`~~

im going to use the ref in mint_validation which revm uses